src/lib.rs

```diff
@@ -859,11 +862,29 @@ impl KeyPair {
 		}
 		Ok(())
 	}
-	/// Serializes the private key in PKCS#8 format
+	/// Return the key pair's public key in DER format
+	///
+	/// The key is formatted according to the SubjectPublicKeyInfo struct of
+	/// X.509 see https://tools.ietf.org/html/rfc5280#section-4.1
+	pub fn public_key_der(&self) -> Vec<u8> {
+		yasna::construct_der(|writer| self.serialize_public_key_der(writer))
+	}
+	/// Return the key pair's public key in PEM format
+	///
+	/// The returned string can be interpreted with `openssl pkey --inform PEM -pubout -pubin -text`
+	#[cfg(feature = "pem")]
+	pub fn public_key_pem(&self) -> String {
+		let p = Pem {
+			tag : "PUBLIC KEY".to_string(),
+			contents : self.public_key_der(),
+		};
+		pem::encode(&p)
+	}
+	/// Serializes the key in PKCS#8 format in DER
```

Comment:
Could you explicitly point out that this includes the private key? The current statement is ambiguous. Either "the private key" or "the key pair" is fine.

Reply:
very good point